main/EventSystem.cpp

```diff
@@ -20,6 +20,9 @@
 #include "../json/json.h"
 #define __STDC_FORMAT_MACROS
 #include <inttypes.h>
+#ifdef __NetBSD__
+#include <machine/int_fmtio.h>
```

Comment:
Why this header?

Reply:
I have no idea. I swear it failed to compile, complaining about text strings that were being built like "...  '%" PRIu64 "' ...", and when I (not having noticed that it would get included anyway) explicitly did include machine/int_fmtio.h, it stopped complaining.

I've removed all those extra includes now, and it's compiling just fine. Of course, I now have GCC 6.4, while I had 5.5 when I made that change, so it's no longer the same environment.

Anyway, thanks to both of you for pointing this out!